concepticondata/conceptlists/Paperno-2019-99.tsv

```diff
@@ -0,0 +1,100 @@
+Paperno-2019-99-26	26	nastojaščij	real	emotional evaluation		
+Paperno-2019-99-27	27	sčastlivyj	happy	emotional evaluation	1495	HAPPY
+Paperno-2019-99-28	28	xorošij	good	emotional evaluation	1035	GOOD
+Paperno-2019-99-29	29	nužnyj	necessary	emotional evaluation	1030	NECESSARY
+Paperno-2019-99-30	30	strašnyj	horrible	emotional evaluation		
+Paperno-2019-99-31	31	jasnyj	clear	emotional evaluation	989	CLEAR
+Paperno-2019-99-32	32	ploxoj	bad	emotional evaluation	1292	BAD
+Paperno-2019-99-33	33	blizkij	close	location properties	3717	NEAR (IN SPACE)
+Paperno-2019-99-34	34	dalekij	faraway	location properties	1406	FAR
+Paperno-2019-99-35	35	tixij	quiet	location properties	1314	QUIET
+Paperno-2019-99-36	36	poslednij	last	order	1529	LAST (FINAL)
+Paperno-2019-99-37	37	sledujuščij	next	order	1354	NEXT
+Paperno-2019-99-38	38	spokojnyj	calm	personality		
+Paperno-2019-99-39	39	uverennyj	confident	personality		
+Paperno-2019-99-40	40	veselyj	funny	personality	1976	MERRY
+Paperno-2019-99-41	41	bol'šoj	big	size	1202	BIG
+Paperno-2019-99-42	42	krupnyj	large	size	1202	BIG
+Paperno-2019-99-43	43	nebol'šoj	small	size	1246	SMALL
+Paperno-2019-99-44	44	ogromnyj	huge	size	1202	BIG
+Paperno-2019-99-45	45	malen'kij	little	size	1246	SMALL
+Paperno-2019-99-46	46	glubokij	deep	depth	1593	DEEP
+Paperno-2019-99-47	47	melkij	shallow	depth	193	SHALLOW
+Paperno-2019-99-48	48	nizkij	low	height	2191	LOW (HEIGHT)
+Paperno-2019-99-49	49	vysokij	high	height	1265	HIGH
+Paperno-2019-99-50	50	dlinnyj	long	length	1203	LONG
+Paperno-2019-99-51	51	korotkij	short	length	1645	SHORT
+Paperno-2019-99-52	52	širokij	wide	width	1243	WIDE
+Paperno-2019-99-53	53	tolstyj	thick	width	1244	THICK
+Paperno-2019-99-54	54	tonkij	thin	width	2249	THIN (OF SHAPE OF OBJECT)
+Paperno-2019-99-55	55	čelovečeskij	human	social value	683	PERSON
+Paperno-2019-99-56	56	strannyj	strange	social value	600	STRANGE
+Paperno-2019-99-57	57	detskij	childish	social value		
+Paperno-2019-99-58	58	osobyj	special	social value		
+Paperno-2019-99-59	59	svobod- nyj	free	social value	2316	FREE
+Paperno-2019-99-60	60	glavnyj	main	social value	1078	MAIN
+Paperno-2019-99-61	61	važnyj	important	social value	1285	IMPORTANT
+Paperno-2019-99-62	62	interesnyj	interesting	social value		
+Paperno-2019-99-63	63	prostoj	simple	social value	686	EASY
+Paperno-2019-99-64	64	velikij	great	social value		
+Paperno-2019-99-65	65	izvestnyj	well-known	social value		
+Paperno-2019-99-66	66	russkij	Russian	social value		
+Paperno-2019-99-67	67	voennyj	military	social value		
+Paperno-2019-99-68	68	ser’eznyj	serious	social value		
+Paperno-2019-99-69	69	živoj	living	social value	1422	BE ALIVE
+Paperno-2019-99-70	70	lučšij	best	social value		
+Paperno-2019-99-71	71	sobstvennyj	own	social value		
+Paperno-2019-99-72	72	ženskij	feminine	social value		
+Paperno-2019-99-73	73	obščij	common	social value		
+Paperno-2019-99-74	74	sovetskij	Soviet	social value		
+Paperno-2019-99-75	75	bystryj	quick	speed	1631	FAST
```

Comment:
bystryj and skoryj differ in meaning: one means fast in form of a movement, and one means fast in form of time succession (first aid is "skoryj", i.e., the help that comes in short time after wards, not the help that runs fast). I'd say: map the more proto-typical one, maybe bystryj, and leave the other one unmapped.